README.md

```diff
@@ -42,12 +42,25 @@ New developers may find the notes in [CONTRIBUTING](https://github.com/JuliaLang
 
 ## Currently Supported Platforms
 
-- **Linux**
-- **MacOS**
-- **FreeBSD**
-- **Windows**
-
-All systems are supported with both x86/64 (64-bit) and x86 (32-bit, except MacOS) architectures. Support for [ARM](https://github.com/JuliaLang/julia/blob/master/README.arm.md), AARCH64, and POWER8 (little-endian) has been added recently.
+Julia is built and tested regularly on the following platforms:
+
+| Operating System | Architecture     | CI | Binaries | Support Level |
+|:----------------:|:----------------:|:--:|:--------:|:-------------:|
+| Linux 2.6.18+    | x86-64 (64-bit)  | ✓  | ✓        | Official      |
+|                  | i686 (32-bit)    | ✓  | ✓        | Official      |
+|                  | PowerPC (64-bit) |    | ✓        | Official      |
+|                  | ARM v7 (32-bit)  |    | ✓        | Official      |
+|                  | AArch64 (64-bit) |    | ✓        | Official      |
```

Comment:
Looks like Yichao is calling it ARM v8 (or rather armv8-a) [here](https://github.com/JuliaLang/julialang.github.com/blob/a1b3255ab9e893fd3d6984db325905c9da894fd9/downloads/index.md#nightly-builds), in his [PR](https://github.com/JuliaLang/julialang.github.com/pull/581) to add it to the downloads page.